### ISSUE TITLE
NameError: uninitialized constant Nistbib

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/nistbib#9]
Urgent bug fix needed:

```
NameError: uninitialized constant Nistbib
Did you mean?  Isobib
               NistBib
  /Users/user/.rbenv/versions/2.5.1/lib/ruby/gems/2.5.0/gems/nistbib-0.1.1/lib/relaton/processor.rb:15:in `get'
```

This is in a released gem.

**Comment by ronaldtse** [comment]
In #10.

**Comment by ronaldtse** [comment]
Fixed in #10.
